.env

```diff
@@ -0,0 +1,3 @@
+# These are defaults and you should change them
```

Comment:
does the compose file automatically pick up this file?

Reply:
yes, tested that

Reply:
It was the recommended way for achieving default values with overidable environment variables

Reply:
You can use `docker compose config` to confirm functionality

Reply:
nice